pkg/converter/convert_unix.go

```diff
@@ -836,6 +873,9 @@ func LayerConvertFunc(opt PackOption) converter.ConvertFunc {
 		// diff id calculation to speed up the conversion.
 		// See: https://github.com/containerd/containerd/blob/e4fefea5544d259177abb85b64e428702ac49c97/images/diffid.go#L49
 		info.Labels[labels.LabelUncompressed] = blobDigest.String()
+		// Update layer labels in remote cache.Actually these two labels will be write to store, too.
+		info.Labels[LayerAnnotationNydusBlob] = "true" 
+		info.Labels[LayerAnnotationSourceDigest] = desc.Digest.String()
```

Comment:
Why do we need to write these labels into content store?

Reply:
Annotation `Actually these two labels will be write to store, too. ` is wrong, in fact, it is not written into the store, it is only written to the cache.